packages/app/src/App.tsx

```diff
@@ -164,11 +163,16 @@ export const App = (): ReactElement => {
     // //         setMetaProvider={setMetaProvider}
     // //     />
     // );
+    const [canSwipe, setCanSwipe] = useState(true);
 
     return (
         <NavigationContainer>
-            {me ? (
-                <>
+            {!props ? (
+                <SafeAreaView style={{ backgroundColor: '#e6ffff', flex: 1 }}>
+                    <Text>Loading</Text>
+                </SafeAreaView>
+            ) : (
+                <TabSwipeContext.Provider value={{ canSwipe, setCanSwipe }}>
```

Comment:
Would this Provider be better suited in Providers.ts? I see you are using state but that should be able to be hoisted up.